xds/internal/client/client_watchers_rds.go

```diff
@@ -22,6 +22,24 @@ import (
 	"time"
 )
 
+// Int64Range is a range for header range match.
+type Int64Range struct {
+	Start int64 `json:"start"`
+	End   int64 `json:"end"`
+}
+
+// HeaderMatcher represents header matchers.
+type HeaderMatcher struct {
+	Name         string      `json:"name,omitempty"`
```

Comment:
Do we need the `omitempty` tag on the `Name` field?

The xDS API has a `min_bytes: 1` annotation on this field:
```
  // Specifies the name of the header in the request.
  string name = 1
      [(validate.rules).string = {min_bytes: 1 well_known_regex: HTTP_HEADER_NAME strict: false}];
```

Reply:
Done